commands/commands.go

```diff
@@ -13,24 +13,26 @@ import (
 	"github.com/skarademir/naturalsort"
 
 	"github.com/docker/machine/drivers"
-	_ "github.com/docker/machine/drivers/amazonec2"
-	_ "github.com/docker/machine/drivers/azure"
-	_ "github.com/docker/machine/drivers/digitalocean"
-	_ "github.com/docker/machine/drivers/exoscale"
-	_ "github.com/docker/machine/drivers/generic"
-	_ "github.com/docker/machine/drivers/google"
-	_ "github.com/docker/machine/drivers/hyperv"
+	//_ "github.com/docker/machine/drivers/amazonec2"
```

Comment:
These naturally will be added back in but are left out to speed up compiles and make the refactor simpler for now.